### PR TITLE
Adds docs

### PR DESCRIPTION
**Pull request by CBroz1**
Despite being part of the google style convention, examples don't render well - they're treated as a markdown text, requiring double line breaks. Open to feedback about how we should navigate

### CODE REVIEW

**Review by kushalbakshi — APPROVED** [review]

File: docs/src/concepts.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## Element Features
+
+### General utilities
+
+`utils.find_full_path` and `utils.find_root_directory` are used 
+across many Elements and Workflows to allow for the flexibility of providing 
+one or more root directories in the user's config, and extrapolating from a relative
+path at runtime.
+
+`utils.ingest_csv_to_table` is used across didactic workflows to ingest from example
```
Comment (dimitri-yatsenko) [inline docs/src/concepts.md]:
```suggestion
`utils.ingest_csv_to_table` is used across workflow examples to ingest from sample data from 
```

Comment (kushalbakshi) [reply docs/src/concepts.md]: Sorry, I thought this was one of my element PRs. Feel free to revert if needed

Comment (CBroz1) [reply docs/src/concepts.md]: No need, I'm on board with this change

File: docs/src/citation.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Citation
+
+If your work uses this Element, please cite the following manuscript and Research
+Resource Identifier (RRID).
+
++ Yatsenko D, Nguyen T, Shen S, Gunalan K, Turner CA, Guzman R, Sasaki M, Sitonic D,
+  Reimer J, Walker EY, Tolias AS. DataJoint Elements: Data Workflows for
+  Neurophysiology. bioRxiv. 2021 Jan 1. doi: https://doi.org/10.1101/2021.03.30.437358
+
++ DataJoint Elements ([RRID:SCR_021894](https://scicrunch.org/resolver/SCR_021894)) -
```
Comment (CBroz1) [inline docs/src/citation.md:10]: Note to self - I need to add [automated versioning to the citation page](https://github.com/datajoint/element-deeplabcut/commit/352c44f98325cb79a249026d84310bff8417e7e2)

**Review by kushalbakshi — APPROVED** [review]

**Review by dimitri-yatsenko — APPROVED** [review]

**Pull request merged**